### DISCUSSION TITLE
Code is updated in master

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#127] [Team Posts]
Okay, everyone, I just pushed all changes from the competition computer to the master branch.

If you have time tomorrow -- please try to review the auton code, with a specific focus on why the Left side auton with right side switch did not work tonight.

I am planning to take a look myself, and I am hopeful that one or more of you will be able to come in tomorrow afternoon with ideas about what's wrong. I'd like to get everything verified and tested ASAP after inspection so we can do some practice matches tomorrow night and hopefully validate that our 2-cube auton works on a real field. 

Thanks again to each and every one of you - I appreciate your hard work and willingness to stay with us into the evening to get things working.

See you all tomorrow!
-Chris

**Comment by coverbeek** [comment]
I just spent a few minutes cleaning up the code to make it readable and I think the issue is pretty easy to find. My hint is that when your indentation is not correct, it is really easy to miss errors in control block scope. I also see a LOT of completely incorrect code documentation, which is more harmful than no code documentation. 

I know we were in a time crunch, but we need to be careful about these types of things because they can lead to exactly the situation we found ourselves in. Work clean and document as you go -- always. Sloppy code leads to bugs.

**Comment by coverbeek** [comment]
Some light reading, if what I said above doesn't make sense to you:
http://icarus.cs.weber.edu/~dab/cs1410/textbook/3.Control/blocks.html
Enjoy!  :)
(or, if you prefer to make your brain hurt, chew on this: https://en.wikipedia.org/wiki/Scope_(computer_science))

**Comment by MikasaSucasa** [comment]
I am glad Chris sees it because I don't after spending 5 minutes on it. ;P
